spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.feature 'User Show', type: :feature do
+  let(:user) { User.create(name: 'Tom', photo: 'https://www.kasandbox.org/programming-images/avatars/leaf-blue.png', bio: 'He is a good programmer') }
+  let!(:post1) { Post.create(author: user, title: 'first post', text: 'first text') }
+  let!(:post2) { Post.create(author: user, title: 'second post', text: 'second text') }
+
+  scenario 'visiting the user Show page' do
+    visit user_path(user)
+
+    expect(page).to have_content('Tom')
+    expect(page).to have_content('Bio')
```

Comment:
- [ ] Dear Team, here for the bio, instead of the text `bio` we are looking for a text which is resined as a user's bio. Please test the bio text instead of just the `bio`.